<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1361</width>
      <height>631</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color:rgb(155, 225, 255);
}


QTabWidget{
 qproperty-icon: url(:/gestion.png);
 qproperty-iconSize: 30px 30px;
text-align:center;
color:black;
background-color:white;
}
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion des patients </string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sexe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>NÂ°telephone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_CIN">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255)
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255)
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255)
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_date">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255)

</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_num">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 qproperty-icon: url(:/ajouter.png);
 qproperty-iconSize: 30px 30px;
text-align:center;
color:black;
background-color:white;
}
QPushButton:pressed{background-color:rgb(68, 206, 101);}</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QTableView" name="tabPatient">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>120</y>
        <width>791</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color:white;

</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>440</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton{
 qproperty-icon: url(:/modifier.png);
 qproperty-iconSize: 30px 30px;
text-align:center;
color:black;
background-color:white;
}
QPushButton:pressed{background-color:rgb(91, 91, 255);}</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>440</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
 qproperty-icon: url(:/supp.png);
 qproperty-iconSize: 30px 30px;
text-align:center;
color:black;
background-color:white;
}
QPushButton:pressed{background-color:rgb(255, 85, 85);}</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_valider">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>360</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton{
 qproperty-icon: url(:/valider.png);
 qproperty-iconSize: 30px 30px;
text-align:center;
color:black;
background-color:white;
}
QPushButton:hover:!pressed{
background-color:rgb(68, 206, 101); }
QPushButton:pressed{background-color:rgb(68, 206, 101);}</string>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_annuler">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>360</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton{
 qproperty-icon:url(:/annuler.png);
 qproperty-iconSize: 30px 30px;
text-align:center;
color:black;
background-color:white;
}
QPushButton:hover:!pressed{
background-color:rgb(255, 85, 85); }
QPushButton:pressed{background-color:rgb(255, 85, 85);}
</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>GESTION DES PATIENTS</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="rd_h">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>441</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid gray;
    border-radius: 5px;
    font-weight: bold;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="le_sexe">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255)

</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>100</y>
        <width>811</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid gray;
    border-radius: 5px;
    font-weight: bold;
}</string>
      </property>
      <property name="title">
       <string>Liste des patients</string>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>320</y>
         <width>311</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_pdf">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>340</y>
         <width>93</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">

QPushButton{
 qproperty-icon: url(:/pdf.png);
 qproperty-iconSize: 30px 30px;
text-align:center;
color:black;
background-color:white;
}
QPushButton:pressed{background-color:rgb(68, 206, 101);}</string>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>451</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid gray;
    border-radius: 5px;
    font-weight: bold;
}
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_cherche">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color:white;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_chercher">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
 qproperty-icon: url(:/chercher.png);
 qproperty-iconSize: 30px 30px;
text-align:center;
color:black;
background-color:white;
}</string>
       </property>
       <property name="text">
        <string>Chercher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_tri">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
 qproperty-icon: url(:/tri.png);
 qproperty-iconSize: 30px 30px;
text-align:center;
color:black;
background-color:white;
}
QPushButton:pressed{background-color:rgb(68, 206, 101);}</string>
       </property>
       <property name="text">
        <string>Tri</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_stat">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>40</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
 qproperty-icon: url(:/stat.png);
 qproperty-iconSize: 30px 30px;
text-align:center;
color:black;
background-color:white;
}
QPushButton:pressed{background-color:rgb(68, 206, 101);}</string>
       </property>
       <property name="text">
        <string>Statistiques</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
image: url(:/gestion.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>groupBox_4</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>rd_h</zorder>
     <zorder>tabPatient</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>le_CIN</zorder>
     <zorder>le_nom</zorder>
     <zorder>le_prenom</zorder>
     <zorder>le_date</zorder>
     <zorder>pb_modifier</zorder>
     <zorder>pb_supprimer</zorder>
     <zorder>pb_valider</zorder>
     <zorder>pb_annuler</zorder>
     <zorder>label_27</zorder>
     <zorder>le_num</zorder>
     <zorder>pb_ajouter</zorder>
     <zorder>label_8</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Statistiques</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>581</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>621</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid gray;
    border-radius: 5px;
    font-weight: bold;
}</string>
      </property>
      <property name="title">
       <string>Statistiques</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>50</y>
        <width>631</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>20</y>
        <width>671</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid gray;
    border-radius: 5px;
    font-weight: bold;
}</string>
      </property>
      <property name="title">
       <string>Statistiques</string>
      </property>
     </widget>
     <zorder>groupBox_5</zorder>
     <zorder>groupBox</zorder>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>verticalLayoutWidget_2</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1351</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
